### PR TITLE
sdk/metric: Fix import comments

### PR DESCRIPTION
**Pull request by andrew-womeldorf**

### CODE REVIEW

**Comment by andrew-womeldorf** [comment]
I went through the EasyCLA workflow to sign on Docusign, but it still shows here as "Missing CLA Authorization". Please let me know if I'm missing something here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Fixed
 
+- Fix inconsistent import comments in sdk/metric
```
Comment (MrAlias) [inline CHANGELOG.md:18]:
```suggestion
- Fix inconsistent import comments in sdk/metric. (#4086)
```

**Comment by MrAlias** [comment]
Your commits look to be missing your email. Please make sure to update your git configuration and re-annotate the commits.

**Review by MrAlias — APPROVED** [review]

**Review by pellared — APPROVED** [review]

**Review by pellared — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

**Pull request merged**